pkg/ccl/importccl/import_processor_test.go

```diff
@@ -914,15 +914,15 @@ func newTestSpec(
 	switch format.Format {
 	case roachpb.IOFileFormat_CSV:
 		descr = descForTable(ctx, t,
-			"CREATE TABLE simple (i INT PRIMARY KEY, s text )", 10, 20, NoFKs)
+			"CREATE TABLE simple (i INT PRIMARY KEY, s text )", 100, 200, NoFKs)
```

Comment:
These descriptor IDs were in the system range and this caused privilege validation failures.